### ISSUE TITLE
Any plans for Jetson AGX Orin support?

### ISSUE THREAD

**Issue by justinblaber** [VC-MIPI-modules/vc_mipi_nvidia#95]
Don't see it explicitly supported here. But this module in particular looks pretty powerful (claims of 275 TOPs) so might potentially enable some more advanced real-time processing that the nanos couldn't handle.

**Comment by lin-voxvision** [comment]
Yes! the same request! please support AGX Orin, thank in advance!
